scan_for_updates.g

```diff
@@ -0,0 +1,66 @@
+#############################################################################
+##
+##  This file is part of GAP, a system for computational discrete algebra.
+##
+##  Copyright of GAP belongs to its developers, whose names are too numerous
+##  to list here. Please refer to the COPYRIGHT file for details.
+##
+##  SPDX-License-Identifier: GPL-2.0-or-later
+##
+
+LoadPackage("json");
+
+InstallMethod(RecNames, [IsRecord and IsInternalRep], x -> AsSSortedList(REC_NAMES(x)));
+
+PackageInfoRec := function(pkginfo_file)
+    Unbind(GAPInfo.PackageInfoCurrent);
+    Read(pkginfo_file);
+    if not IsBound(GAPInfo.PackageInfoCurrent) then
+      # TODO better error message
+      Error(StringFormatted("reading {} failed", pkginfo_file));
+    fi;
+    return GAPInfo.PackageInfoCurrent;
+end;
+
+InstallMethod(_GapToJsonStreamInternal,
+[IsOutputStream, IsObject],
+function(o, x)
+    PrintTo(o, "null");
+end);
+
+WriteJson := function(json_dir, pkginfo_record)
+  local json_fname, root;
+  # TODO is the following necessary?
```

Comment:
Well, we don't want to put absolute paths into the file.

But maybe you are asking if we really need to store `InstallationPath` itself: currently we do (e.g. the website needs it to adjust manual paths). But of course our plan is to change the package distribution so that all packages will be in `pkg/lowercase_package_name`, then it can be derived....

BUT for a simpler transition, it would be really helpful to keep this in for now. The fewer things we have to change elsewhere, e.g. on the website, the better... Once the dust has settled, we can start a big cleanup and get rid of things like this, if we feel like it.